EvaluableExpression.go

```diff
@@ -207,3 +207,16 @@ func (this EvaluableExpression) String() string {
 
 	return this.inputExpression
 }
+
+/*
+	Returns an array representing the variables contained in this EvaluableExpression.
+*/
+func (this EvaluableExpression) Vars() []string {
+	var varlist []string
+	for _, val := range this.Tokens() {
+		if val.Kind == 7 {
```

Comment:
It'd be nice to see this changed to `VARIABLE` (as defined in TokenKind.go) instead of `7`. I doubt it will ever change, but if it does, it'll be nice to not have to chase down what `7` used to mean.